try {
  throw new Error();
} catch (error) {
  console.log(error);
}

const {} = {};
